src/ui/components/ShowMoreCard/index.js

```diff
@@ -55,6 +60,10 @@ export class ShowMoreCardBase extends React.Component {
         <a className="ShowMoreCard-revealMoreLink" href="#show-more"
           onClick={this.expandText}>
           {i18n.gettext('Show more info…')}
```

Comment:
It looks like Facebook has a "Continue Reading" link on long posts in their Android app. I'm not sure if there is any extra context provided on Android but on the desktop web it just shows "Read More".

Not saying we should necessarily do what they are but it could be that these sort of links are commonplace and adding more context might not make sense. I could see if you had something like "**Read more** of Alice's article" that would be bad.

I'm not sure what the solution is but this feels too verbose and "Read More" alone does feel like without context you couldn't tell if the site was not screen reader friendly or if it was one of these "expand section" links.

`<span className="visually-hidden">Expand to</span> Read More`?